staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -54,15 +54,13 @@ func NewServerRunOptions() *ServerRunOptions {
 }
 
 // ApplyOptions applies the run options to the method receiver and returns self
-func (s *ServerRunOptions) ApplyTo(c *server.Config) error {
```

Comment:
I'd like to keep this for congruence.  Without it, the code looks bugged with respect to all of its peers.